ddtrace/llmobs/_integrations/base.py

```diff
@@ -81,18 +85,15 @@ def llmobs_enabled(self) -> bool:
         return LLMObs.enabled
 
     def is_pc_sampled_span(self, span: Span) -> bool:
-        if span.context.sampling_priority is not None:
-            if span.context.sampling_priority <= 0:
-                return False
+        if span.context.sampling_priority and span.context.sampling_priority <= 0:
```

Comment:
Assuming we don't want this to always evaluate to false if sampling priority is zero
```suggestion
        if span.context.sampling_priority is not None and span.context.sampling_priority <= 0:
```

Reply:
We only care about positive non-zero sampling priority values here, even if `span.context.sampling_priority == 0`, the first conditional overlaps with the other conditional `span.context.sampling_priority <= 0` so this should be fine

Reply:
Nevermind, missed what you were saying. Thanks!